interactions/api/cache.py

```diff
@@ -12,6 +12,8 @@
     overload,
 )
 
+import interactions
```

Comment:
why are you doing this?

Reply:
It avoids a circular import for the LRUDict

Reply:
Alright